cfgov/jinja2/v1/_includes/organisms/full-width-text.html

```diff
@@ -30,7 +30,7 @@
                 {% set media = image(block.value, 'original') %}
                 <img class="m-full-width-media_image"
                      src="{{ media.url }}"
-                     alt="{{ block.value.alt or media.alt }}">
+                     alt="{{ media.alt }}">
```

Comment:
Because it's not an `ImageBasic` block, so you can't pass the right thing into the tag.